### PR TITLE
fix: set document overflow style to prevent screen jump

### PR DESCRIPTION
**Pull request by AlinaGovoruhina**
# Issue

#338 

## Description

Set document overflow to auto when dialog is open

## Checklist for Pull Requests

- [x] Supplied as many details as possible on this change
- [x] Included the link to the associated issue in the Issue section above
- [x] The code conforms to the general [development guidelines](https://github.com/SAP/guided-answers-extension/blob/main/docs/developer-guide.md).
- [ ] The code has **unit tests** where applicable and is easily unit-testable
- [x] This branch is appropriately named for the associated issue

### CODE REVIEW

**Review by hitesh-parmar — CHANGES_REQUESTED** [review]
Hi @AlinaGovoruhina, could you please create a changeset for a patch update as well?
I guess the command is `pnpm changeset`

**Review by Klaus-Keller — APPROVED** [review]
Thanks @AlinaGovoruhina!
- css changes look good
- change set exists
- did a local test and could not find any issues

Approved from my side.

File: .changeset/rare-parents-press.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@sap/guided-answers-extension-webapp": patch
```
Comment (hitesh-parmar) [inline .changeset/rare-parents-press.md:2]: Hi @Klaus-Keller, shouldn't changeset include patch update of all the modules?

Comment (Klaus-Keller) [reply .changeset/rare-parents-press.md:2]: @hitesh-parmar, this should take care of it:
https://github.com/SAP/guided-answers-extension/blob/84f6c1da168eac06dc0ada5d9f8c4fc651f7dc6f/.changeset/config.json#L5

Comment (hitesh-parmar) [reply .changeset/rare-parents-press.md:2]: Thanks Klaus

**Review by hitesh-parmar — APPROVED** [review]

File: .changeset/rare-parents-press.md (hunk 1 above)
Comment (hitesh-parmar) [reply .changeset/rare-parents-press.md:2]: Hi @Klaus-Keller , Looks like in this case changelog entry is not created https://github.com/SAP/guided-answers-extension/blob/main/packages/ide-extension/CHANGELOG.md

**Pull request merged**